omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -207,40 +211,46 @@ OMERO configuration
 
     $ open -a TextEdit.app ~/.bash_profile
 
-   Add an environment variable :envvar:`OMERO_SERVER` to the :file:`.bash_profile` which points
+   Add an environment variable :envvar:`OMERODIR` to the :file:`.bash_profile` which points
    to the location of the OMERO executable::
 
     # Pre-built server...
-    export OMERO_SERVER=/path/to/OMERO.server-x.x.x-ice36-bxx
+    export OMERODIR=/path/to/OMERO.server-x.x.x-ice36-bxx
     # ...OR locally built server
-    export OMERO_SERVER=/path/to/openmicroscopy/dist
+    export OMERODIR=/path/to/openmicroscopy/dist
 
    and add the OMERO executable to the OS X :envvar:`PATH`::
 
     # Add the OMERO distribution to PATH
-    export PATH=$OMERO_SERVER/bin:$PATH
+    export PATH=$OMERODIR/bin:$PATH
```

Comment:
Do we need to do this now, since in the virtualenv we will have ```omero``` available already?